.github/workflows/test-python.yml

```diff
@@ -33,7 +33,7 @@ jobs:
         uses: actions/checkout@v4
         with:
           repository: 'mongodb-forks/django'
-          ref: 'mongodb-5.1.x'
+          ref: 'objectid-no-int'
```

Comment:
will this branch get merged into both 5.1 and 5.0?

Reply:
Of course. Whenever you see a change like this is just means we needed to use a temporary staging branch until the patch is merged.